Dockerfile

```diff
@@ -18,11 +18,14 @@ COPY vendor/ vendor/
 RUN GODEBUG=madvdontneed=1 GOGC=50 CGO_ENABLED=1 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o build/manager cmd/manager/main.go
 RUN GODEBUG=madvdontneed=1 GOGC=50 CGO_ENABLED=1 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o build/server cmd/server/main.go
 
-FROM registry.access.redhat.com/ubi9/ubi-minimal:9.4
+
+FROM registry.ci.openshift.org/ocp/4.17:base-rhel9
 
 ARG DATA_DIR=/data
 RUN mkdir $DATA_DIR && chmod 775 $DATA_DIR
 
+RUN dnf install -y nmstate-libs nmstate && dnf clean all && rm -rf /var/cache/dnf/*
```

Comment:
Specify the nmstate version you wish to install, this effects the nmstate API we support (e.g. mac-identifier) so we should probably document it)